### PR TITLE
New Feature: Time Format Customization

### LINKED ISSUE #132: Only show date
Maybe it is a nice feature to toggle if you want to show the time, since the time is frozen quite often.

### PR DESCRIPTION
**Pull request by L1cardo**
---
name: add new feature
about: Create a pull request to contribute in making Pock amazing!
title: 'add new feature'
labels: ''
assignees: ''

---

**Describe the Pull request**
1. add time format customization. this will fix time format issue in some regions. For example in China ,we use `year month day time` instead of `year day month time`
截屏2019-10-12下午4 34 19
2. add simpilified Chinese readme
3. update some simpilified Chinese localization
4. nowplaying supports google chrome

**Screenshots**
Time format：
image
image

An interesting feature：
it supports emoji！
image

**Fixes**
fix some region time format issue
fix #132 

**Pull request type**
Keep only the option that better matches your pull request.

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

**Tested?**
Yes

*Test Configuration*:
* Hardware:      MacBook Pro (13-inch, 2018, Four Thunderbolt 3 Ports)
* macOS Version: 10.15
* Xcode version: 11.1

**Checklist**
Use this list to keep track of your progress:

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by L1cardo** [comment]
Time format textfield can live check textchange now, we don't need OK button anymore!
This is what it looks like now!

image

**Comment by L1cardo** [comment]
demo

File: Pock/Preferences/Panes/GeneralPreferencePane/GeneralPreferencePane.swift [hunk 1]
```diff
@@ -63,9 +63,9 @@ final class GeneralPreferencePane: NSViewController, PreferencePane {
     }
     
     private func setupCheckboxes() {
-        self.hideControlStripCheckbox.state = Defaults[.hideControlStrip]         ? .on : .off
-        self.enableAutomaticUpdates.state   = Defaults[.enableAutomaticUpdates]   ? .on : .off
-        self.launchAtLoginCheckbox.state    = Defaults[.launchAtLogin]            ? .on : .off
+        self.hideControlStripCheckbox.state = Defaults[.hideControlStrip]              ? .on : .off
+        self.enableAutomaticUpdates.state   = Defaults[.enableAutomaticUpdates]        ? .on : .off
+        self.launchAtLoginCheckbox.state    = LoginServiceKit.isExistLoginItems()      ? .on : .off
```
Comment (pigigaldi) [inline Pock/Preferences/Panes/GeneralPreferencePane/GeneralPreferencePane.swift]: 👌

Comment (L1cardo) [reply Pock/Preferences/Panes/GeneralPreferencePane/GeneralPreferencePane.swift]: > 👌

Yes， hahaha
I noticed that and have fixed it

**Comment by pigigaldi** [comment]
Merged and included in 0.6.3
Thanks, @L1cardo 🎉

**Comment by L1cardo** [comment]
@pigigaldi 
Haha  you’re welcome!
You may want to change the url of time format help button, I didn’t find a proper website to let time formatting easy and clear for users, so I just added a url of another app.
I don’t know if it is OK to open another app’s website when using Pock.

**Pull request closed without merging**